Client/Frontend/Widgets/SiteTableViewController.swift

```diff
@@ -93,8 +93,8 @@ class SiteTableViewController: UIViewController, UITableViewDelegate, UITableVie
 
         tableView.delegate = self
         tableView.dataSource = self
-        tableView.registerClass(TwoLineTableViewCell.self, forCellReuseIdentifier: CellIdentifier)
-        tableView.registerClass(SiteTableViewHeader.self, forHeaderFooterViewReuseIdentifier: HeaderIdentifier)
```

Comment:
Is `SiteTableViewHeader` unused now? I don't see it being used in `RemoteTabsPanel`, either. If so, just put your `HistoryTableViewHeader` changes directly into `SiteTableViewHeader`?